esp-wifi/src/ble/npl.rs

```diff
@@ -711,14 +698,12 @@ unsafe extern "C" fn ble_npl_callout_deinit(callout: *const ble_npl_callout) {
 unsafe extern "C" fn ble_npl_callout_stop(callout: *const ble_npl_callout) {
     trace!("ble_npl_callout_stop {:?}", callout);
 
-    if (*callout).dummy == 0 {
-        panic!("Trying to stop an uninitialzed callout");
-    }
+    core::assert!((*callout).dummy != 0);
```

Comment:
Any reason these aren't the assert from `fmt.rs`? I'd prefer using defmt if it's enabled.